### ISSUE TITLE
navigation.state is undefined when navigating back from materialTopTabNavigatior

### ISSUE THREAD

**Issue by TheTimeWalker** [IjzerenHein/react-navigation-shared-element#26]
When a `sharedElementStack` has a nested materialTopTabNavigator inside as a screen, an exception `TypeError: undefined is not an object (evaluating 'navigation.state')` pops up when trying to go back from it. An example:
```javascript
const TabScreen = createMaterialTopTabNavigator(
  {
    InsideOne: { screen: InsideOne },
    InsideTwo: { screen: InsideTwo },
  }
)

const SharedStack = createSharedElementStackNavigator(
  {
    Main: { screen: MainScreen },
    TabScreen: { screen: TabScreen }
  }
)
```
When navigating to the TabScreen and then navigating back by tapping on the back arrow, the app crashes as soon as the transition is done.

**Comment by IjzerenHein** [comment]
I've fixed the issue and release a new version 👍 
https://github.com/IjzerenHein/react-navigation-shared-element/releases/tag/v2.1.0

**Comment by IjzerenHein** [comment]
Can I close this issue?

**Comment by TheTimeWalker** [comment]
I couldn't test it out yet, sorry! Can you give me just a tiny bit? I'll close it if everything works

**Comment by IjzerenHein** [comment]
Yeah of course, I'll leave this open and up to you to close this then 👍

**Comment by TheTimeWalker** [comment]
I had two reproducible cases where it crashed and tested it out quickly. It seems like this is now both fixed with 2.1.0, thank you!

**Comment by IjzerenHein** [comment]
Awesome!
